src/makeCalculator.js

```diff
@@ -37,7 +37,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, number) {
+      this.result = callback(this.result, number);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(augend, addend) {
+      return augend + addend;
+    },
```

Comment:
you can use result from your calculator

```suggestion
    add(addend) {
      return calculator.result + addend;
    },
```

Reply:
I tried to do it like this:
```
add(addend) {
      this.result += addend;
    },
```
But it didn`t work, so i thought that it would be a good idea to put result like an argument of function. I forgot that _object._  could be used instead of _this._

Reply:
Need more read about this )
When you use constructors, classes, or something like that `this` looks to service which call your method
`calculator.reset()` -> this on reset is a calculator
But `add` is a callback, the function doesn’t have this:
`add()` -> this isn’t exit here